laboratorium/przykładowy.c

```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
+
+float main() {
+float a;
+scanf("%f",&a);
+printf("%f\n",a);
+
+float b;
+scanf("%f",&b);
+printf("%f\n",b);
+
+float c;
+scanf("%f",&c);
+printf("%f\n",c);
+
+float wynik = (a*b) - (b/c);
+printf("%f\n",wynik);
+
+
+
+
+
+
+return 0;
+int a,b,c;
+
+if ((a>b) && (b>c) && (a>c)) {
```

Comment:
jezeli **a** jest wieksze od **b** oraz **b** jest wieksze od **c** - to **a** musi być większe od **c**.
Ostatni warunek zbędny.